example/simple/src/main.c

```diff
@@ -91,7 +91,8 @@ static void simple_complete(ocf_cache_t cache, void *priv, int error)
  */
 int initialize_cache(ocf_ctx_t ctx, ocf_cache_t *cache)
 {
-	struct ocf_mngt_cache_config cache_cfg = { .name = "cache1" };
+	struct ocf_mngt_cache_config cache_cfg = { .name = "cache1",
+											   .name_len = sizeof("cache1") };
```

Comment:
in editor this line starts under '.name'

Reply:
So probably you have tab length set to 4 chars instead of 8.

Reply:
that was true